client-ts/Microsoft.AspNetCore.SignalR.Test.Server/wwwroot/connectionTests.html

```diff
@@ -18,9 +30,10 @@
         }
         else
         {
+            var minified = getParameterByName('debug') !== 'true' ? '.min' : '';
```

Comment:
Shouldn't this be applied to ES5 too

Reply:
OK (didn't do it because you would have to debug in IE which we typically don't do but it does not cost much and you may hit an IE specific issue...)